core/sector_storage/impl/scheduler_impl.hpp

```diff
@@ -56,12 +56,8 @@ namespace fc::sector_storage {
   };
 
   inline bool operator<(const TaskRequest &lhs, const TaskRequest &rhs) {
-    return less(rhs.priority,
-                lhs.priority,
-                lhs.task_type,
-                rhs.task_type,
-                lhs.sector.id.sector,
-                rhs.sector.id.sector);
+    return std::tie(rhs.priority, lhs.task_type, lhs.sector.id.sector)
+           < std::tie(lhs.priority, rhs.task_type, rhs.sector.id.sector);
```

Comment:
```suggestion
    return std::tie(lhs.priority, lhs.task_type, lhs.sector.id.sector)
           < std::tie(rhs.priority, rhs.task_type, rhs.sector.id.sector);
```
Confused about `priority`.

Reply:
It is correct.
If you have a higher priority then you should be less. Because we use this operator in `set`.

Reply:
https://github.com/filecoin-project/lotus/blob/master/extern/sector-storage/request_queue.go#L16